openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/CollectionDAO.java

```diff
@@ -4561,11 +4552,11 @@ List<String> getTagsStartingWithPrefix(
   interface TagUsageDAO {
     @ConnectionAwareSqlUpdate(
         value =
-            "INSERT IGNORE INTO tag_usage (source, tagFQN, tagFQNHash, targetFQNHash, labelType, state, reason) VALUES (:source, :tagFQN, :tagFQNHash, :targetFQNHash, :labelType, :state, :reason)",
+            "INSERT IGNORE INTO tag_usage (source, tagFQN, tagFQNHash, targetFQNHash, labelType, state, reason, appliedBy) VALUES (:source, :tagFQN, :tagFQNHash, :targetFQNHash, :labelType, :state, :reason, :appliedBy)",
```

Comment:
applied By is not mandatorily available while checking Table tags for example we want to be able to list all the tags that asset has